Resources/Prototypes/_Sunrise/Entities/Objects/Fun/figurines.yml

```
@@ -21,3 +21,51 @@
+    sprite: _Sunrise/Objects/Fun/figurines.rsi
+    state: brigmedic
+  - type: SpeakOnTrigger
+    pack: FigurinesBrigmedic
+
+- type: entity
+  parent: BaseFigurine
+  id: ToyFigurineCburn
+  name: Cburn figure
+  description: A figurine depicting the station's defenders against... viruses?
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)